### ISSUE TITLE
Bug: The SoundBlaster mixer has no effect

### ISSUE THREAD

**Issue by maximilien-noal** [OpenRakis/Spice86#357]
**Describe the bug**

The SoundBlaster mixer has no effect.

**To Reproduce**
1. Go to Dune's mixer settings
2. Click on the PCM volume or MIDI volume. Lower it, or make it go up, or modify the stereo panning. 
3. See that there is no effect whatsoever on the emulator's audio.

**Expected behavior**

The SoundBlaster mixer should affect the sofware mixer, and in a audible way.

**Screenshots**

![image](https://github.com/OpenRakis/Spice86/assets/1087524/f2219b92-c3d1-48d6-9765-bc8957db9d91)

**Desktop:**
 - OS: Fedora Workstation
 - Version: latest master

**Additional context**

For starters, the AudioPlayer usage should be centralized.
Then the mixer should accept channels as input, and handle emulation pause and continue.
Next, it should react to the SoundBlaster mixer.
Basically, everything that the DOSBox mixer does, we do it too, when applicable.
But, based on PortAudio.
Using the PortMixer library may help.
